manifests/components/04d_argocd-server-deployment.yaml

```diff
@@ -33,6 +33,22 @@ spec:
         volumeMounts:
         - mountPath: /shared
           name: static-files
+
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+          initialDelaySeconds: 2
+          periodSeconds: 1
+          failureThreshold: 30
```

Comment:
My understanding of readiness was wrong. Readiness is not just during server startup. It applies throughout the lifetime of the pod, so we cannot be so aggressive. Lets remove liveness entirely and simply have readiness with the following settings:
```
        readinessProbe:
          httpGet:
            path: /healthz
            port: 8080
          initialDelaySeconds: 3
          periodSeconds: 30
```